client/apollo/js/View/Track/AnnotTrack.js

```diff
@@ -2294,6 +2331,71 @@ define([
                     'class': "annotation_info_editor_button"
                 }, commentButtons);
 
+                var replacementDiv = dojo.create("div", {'class': "annotation_info_editor_section"}, content);
+                var replacementLabel = dojo.create("div", {
+                    'class': "annotation_info_editor_section_header",
+                    innerHTML: "Replace model(s)"
+                }, replacementDiv);
+
+             
+                var replacementsTable = dojo.create("div", {
+                    'class': "replacement",
+                    id: "replacement_" + (selector ? "child" : "parent")
+                }, replacementDiv);
+                var replacementButtonsContainer = dojo.create("div", {style: "text-align: center;"}, replacementDiv);
+                var replacementButtons = dojo.create("div", {'class': "annotation_info_editor_button_group"}, replacementButtonsContainer);
+                var addreplacementButton = dojo.create("button", {
+                    innerHTML: "Add",
+                    'class': "annotation_info_editor_button"
+                }, replacementButtons);
+                var deletereplacementButton = dojo.create("button", {
```

Comment:
It seems to add the attribute properly, so I would probably get rid of the "delete" portion for now, since you can just delete it from the attributes.  You can always add it later.